tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Get Calculations.feature

```diff
@@ -76,3 +77,25 @@ Scenario: verify the price calculations for an order with multiple order items
         | AdditionalService | Declarative      | Month    | 9000  | 1        | Year             | 11600                  | 139200                | 200             | 417800             |
         | Solution          | Patient          | Year     | 450   | 3        | Month            | 2712.5                 | 32550                 | 200             | 97850              |
         | AssociatedService | Declarative      | NULL     | 125   | 100      | NULL             | 2600                   | 31200                 | 12700           | 106300             |
+
+Scenario: verify the price calculations for Catalogue Solution order recipients
+    Given catalogue items exist
+        | Id        | Name | CatalogueItemType |
+        | 10001-001 | Sol1 | Solution          |
+    And order items exist
+        | OrderId | CatalogueItemId | Price   | ProvisioningType   | PriceTimeUnit |
+        | 10001   | 10001-001       | 5       | Patient            | Year          |
+    And order item recipients exist
+        | OrderId | CatalogueItemId | OdsCode   | Quantity   |
+        | 10001   | 10001-001       | eu        | 10         |
+        | 10001   | 10001-001       | au        | 20         |
+    And the user creates a request to retrieve the details of an order by ID 10001
+    When the user sends the get order request
+    Then a response with status code 200 is returned
+    And the get order response contains recipient with <OdsCode> a yearly value of <RecipientTotal>

```

Comment:
I think this might be better if it was something like `And the get order response contains a recipient with <OdsCode> for catalogue item ID [ID-here] and a yearly value of <RecipientTotal>`.

We could then filter by catalogue ID.

What do you think?

Reply:
Yes that will add filter functionality on `CatalogueItemId` level.
I have rephrased scenario step and also updated steps accordingly.